internal/fakecgo/trampolines_amd64.s

```diff
@@ -73,11 +74,21 @@ TEXT ·setg_trampoline(SB), NOSPLIT, $0-16
 	CALL BX
 	RET
 
-TEXT threadentry_trampoline(SB), NOSPLIT, $16
+TEXT threadentry_trampoline(SB), NOSPLIT, $0
+	// See crosscall2.
+	PUSH_REGS_HOST_TO_ABI0()
+
+	// X15 is designated by Go as a fixed zero register.
+	// We are calling directly into ABIInternal,
+	// so we need to ensure it's zero.
+	PXOR	X15, X15
```

Comment:
Use space instead of tab for consistency (but is a tab more conventional for Go?)

Reply:
Hmm I think tabs are more standard. If ran through [asmfmt](https://github.com/klauspost/asmfmt) it uses tabs. Perhaps we should standardize on that. Tabs for indentation and spaces for alignment is the same for gofmt